.prettierrc.js

```diff
@@ -9,22 +9,4 @@ module.exports = {
     require("prettier-plugin-packagejson"),
     require("prettier-plugin-sh"),
   ],
-  overrides: [
-    {
-      files: [".eslintrc.cjs"],
-      options: {
-        quoteProps: "preserve",
-        semi: false,
-        trailingComma: "none",
-      },
-    },
-    {
-      files: ["blocks/**/.eslintrc.cjs"],
-      options: {
-        quoteProps: "as-needed",
-        semi: true,
-        trailingComma: "all",
-      },
-    },
-  ],
```

Comment:
Negates diff described in https://github.com/hashintel/hash/pull/1306#discussion_r1010421641